### DISCUSSION TITLE
Adding More Than one Device to a Single Integration?

### DISCUSSION THREAD

**Discussion by dfayne** [sbidy/wiz_light#202] [General]
Sorry if this is a dumb question but is there a way to add more than one device to a single integration? I couldn't find a way so I just added a new integration every time I wanted to add a bulb to HA. This is what my Integration window looks like. Is this correct? 
<img width="332" alt="image" src="https://user-images.githubusercontent.com/93413544/142340635-99192a5a-a669-483a-9725-7975fc6a9cef.png">

**Comment by weberjam007** [comment]
Hi @dfayne 👋🏼

Your integration window does look correct.

@sbidy  might be able to add deeper details here, but some integrations have an instance of the integration and a list of devices that use that integration (like you have pictured above).  I think that is because each one of the WiZ Lights is a separate device with its own IP address and thus needs its own instance of the integration running in the background to connect each light to the system and manage that control and monitoring message flow.

Other integrations have a summary of devices/services and entities for that integration (look at the "Supervisor" entity if you it).  But I believe if you have two physical devices you would have the dialog look like your WiZ dialog. E.g. if you add one smart thermostat, you will get a summary of the devices connected to the thermostat (the thermostat itself and any smart sensor that the thermostat manages). If you add a second smart thermostat of the exact same type, then the dialog would change to have two lines just like the WiZ dialog.
